packages/core/useDebounce/index.stories.tsx

```diff
@@ -1,52 +1,37 @@
-import 'vue-tsx-support/enable-check'
-import Vue from 'vue'
-import { storiesOf } from '@storybook/vue'
+import { defineDemo, html } from '../../_docs'
 import { defineComponent, ref, watch } from 'vue-demi'
-import { ShowDocs } from '../../_docs/showdocs'
 import { useDebounce } from '.'
 
-type Inject = {
-  input: string
-  debounced: string
-  updated: number
-}
-
-const Demo = defineComponent({
-  setup() {
-    const input = ref('')
-    const debounced = useDebounce(input, 1000)
-    const updated = ref(0)
-
-    watch(debounced, () => {
-      updated.value += 1
-    }, { lazy: true })
-
-    return {
-      input,
-      debounced,
-      updated,
-    }
+defineDemo(
+  {
+    name: 'useDebounce',
+    category: 'Utilities',
+    docs: require('./index.md'),
+    module,
   },
-
-  render(this: Vue & Inject) {
-    // @ts-ignore
-    const Docs = <ShowDocs md={require('./index.md')} />
-
-    return (
+  defineComponent({
+    setup() {
+      const input = ref('')
+      const debounced = useDebounce(input, 1000)
+      const updated = ref(0)
+
+      watch(debounced, () => updated.value += 1)
+
+      return {
+        input,
+        debounced,
+        updated,
+      }
+    },
+
+    template: html`
       <div>
-        <div id="demo">
-          <input v-model={this.input} placeholder="Try to type anything..."/>
+          <input v-model="input"" placeholder="Try to type anything..."/>
           <note>Delay is set to 1000ms for this demo.</note>
 
-          <p>Debounced: {this.debounced}</p>
-          <p>Times Updated: {this.updated}</p>
-
-        </div>
-        {Docs}
+          <p>Debounced: {{this.debounced}}</p>
+          <p>Times Updated: {{this.updated}}</p>
```

Comment:
```suggestion
          <p>Debounced: {{debounced}}</p>
          <p>Times Updated: {{updated}}</p>
```

Reply:
Sorry, I noticed these flaws. I'm more careful now. I am testing all functions before changing and after changing to ensure that it will work as expected.